karma.conf.js

```diff
@@ -130,8 +135,8 @@ module.exports = function (config) {
     captureTimeout: 120000,
     colors: true,
     logLevel: config.LOG_INFO,
-    autoWatch: false,
-    singleRun: true,
+    autoWatch: true,
```

Comment:
Did you mean to commit these changes? I don't think the CI build will work with these settings.